third_party/move/move-model/src/ast.rs

```diff
@@ -2477,6 +2487,33 @@ impl Operation {
     pub fn allows_ref_param_for_value(&self) -> bool {
         matches!(self, Operation::Eq | Operation::Neq)
     }
+
+    /// Get the string representation, if this is a binary operator.
+    /// Returns `None` for non-binary operators.
+    pub fn to_string_if_binop(&self) -> Option<String> {
```

Comment:
You should return an `Option<&'static str>` so we don't unnecessarily create a lot of identical strings (note this is called proactively on every operation, not just when there is an error).

Reply:
done